code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -96,6 +96,9 @@
 
 	var/hivemind_countdown = SSticker.mode?.get_hivemind_collapse_countdown()
 	.["hive_orphan_collapse"] = !isnull(hivemind_countdown) ? hivemind_countdown : 0
+	var/datum/game_mode/infestation/nuclear_war/mode = SSticker.mode
+	var/siloless_countdown = mode.get_siloless_collapse_countdown()
+	.["hive_silo_collapse"] = !isnull(siloless_countdown) ? siloless_countdown : 0
```

Comment:
Honestly would probably go back and make it on /gamemode because if its not NW this will runtime (e.g testing enviroment with extended) or you need to typecheck every instance where these procs are accessed

Open to other maints input though